### PR TITLE
Highscores increment fix

### PR DESCRIPTION
**Pull request by daanwerf**
Fixed the bug where the highscores would not increment after restarting the game once or more. The bug was solved by omitting the time penalty from the equation. Also removed enemy 12 from the game.

### CODE REVIEW

File: src/main/java/lasersharks/behaviour/highscoreincrement/FishHighScoreIncrementBehaviour.java [hunk 1]
```diff
@@ -21,7 +21,7 @@ public FishHighScoreIncrementBehaviour(final Displayable element) {
   }
 
   @Override
-  public int onCollisionHighScoreIncrement(final int timePenalty) {
-    return (int) Math.max(element.getSize() + element.getSpeed() - timePenalty, 0);
+  public int onCollisionHighScoreIncrement() {
+    return (int) Math.max(element.getSize() + element.getSpeed() - element.getSize(), 0);
```
Comment (Exclaminator) [inline src/main/java/lasersharks/behaviour/highscoreincrement/FishHighScoreIncrementBehaviour.java]: Size + speed - size = speed

File: src/main/java/lasersharks/behaviour/highscoreincrement/DefaultHighScoreIncrementBehaviour.java [hunk 2]
```diff
@@ -3,18 +3,16 @@
 import lasersharks.behaviour.HighScoreIncrementBehaviour;
 
 /**
- * High score Increment behaviour for SeaObjects.
- <<<<<<< HEAD
- =======
- * @author SEMGroup27
->>>>>>> remotes/origin/master
+ * High score Increment behaviour for SeaObjects. <<<<<<< HEAD =======
```
Comment (SytzeAndr) [inline src/main/java/lasersharks/behaviour/highscoreincrement/DefaultHighScoreIncrementBehaviour.java]: the merge conflict generated things that are now in javadoc should be removed.

**Comment by SytzeAndr** [comment]
looks good. I'm having problems locally with running our program so I couldn't test it. If the bugs are resolved and it works, I'd like to see this merged.

**Comment by MichielvdBerg** [comment]
I approve

**Comment by spiderbiggen** [comment]
If you could move the location of the indicator to the top right then that would be great

**Pull request merged**